components/perl/XSLoader/pkg5

```diff
@@ -0,0 +1,14 @@
+{
+    "dependencies": [
+        "SUNWcs",
+        "runtime/perl-522",
+        "runtime/perl-524",
+        "runtime/perl-534",
+        "shell/ksh93",
+        "system/library"
+    ],
+    "fmris": [
+        "library/perl-5/xsloader",
+    ],
```

Comment:
And again, here are missing 3 fmris because of missing -$(PLV) in the manifest.

Reply:
Inherited fix